File: stress/t/flt_accum.f
      PROGRAM FACCUM
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 I,N
      N=1000
      ACC=0.0D0
      DO 10 I=1,N-1
        ACC=ACC+DBLE(I)*0.001D0
   10 CONTINUE
      WRITE(*,'(A,F0.6)') 'acc= ',ACC
      END

File: stress/t/flt_accum.c
#include <stdio.h>
#include <math.h>
int main(void){
  long i,n=1000; double acc=0.0;
  for(i=1;i<n;i++){ acc=acc+(double)i*0.001; }
  printf("acc= %f\n",acc);
  return 0;
}
